cpp/daal/include/algorithms/covariance/covariance_batch.h

```diff
@@ -271,6 +271,20 @@ class DAAL_EXPORT BatchImpl : public daal::algorithms::Analysis<batch>
         _hpar = other.daal::algorithms::Analysis<batch>::_hpar;
     }
 
+    /**
+     * Copy-assignment operator for an algorithm for correlation or variance-covariance matrix computation
+     * \param[in] other An algorithm to be used as the source to initialize the input objects
+     *                  and parameters of the algorithm
+     */
+    BatchImpl & operator=(const BatchImpl & other)
```

Comment:
@Vika-F Any way to verify for sure whether this could potentially cause 'illegal instruction' in AVX2? It seems to be an exported class though so I'm thinking it might not undergo the same multi-compilation process as the others.

Reply:
The thing is that I can't find the respective cpu files at all, and since the implementation of other functions was done in header I also added my implementation to header

Reply:
Yes, looks like it shouldn't be a problem if there's already other implementations in the same file calling the same methods.